scan/src/components/modal/ValidatorStakingInfo.re

```diff
@@ -134,18 +143,28 @@ module StakingInfo = {
         dispatchModal(OpenModal(SubmitTx(SubmitMsg.Undelegate(validatorAddress))));
       let withdrawReward = () =>
         dispatchModal(OpenModal(SubmitTx(SubmitMsg.WithdrawReward(validatorAddress))));
+      let isReachUnbondingLimit = unbondingList |> Belt_Array.length == 7;
+
       <div>
         <VSpacing size=Spacing.md />
         {rewardAmount.amount > 1.
            ? <div>
-               <div className=Styles.warning>
+               <div className=Styles.reminder>
                  <Text
                    value="Note: You have non-zero pending reward on this validator. Any additional staking actions will automatically withdraw that reward your balance."
                  />
                </div>
                <VSpacing size=Spacing.lg />
              </div>
            : React.null}
+        {isReachUnbondingLimit
+           ? <div>
+               <div className=Styles.warning>
+                 <Text value="Warning: You have reached the unbonding count. (7 times)" />
```

Comment:
Change to "Warning: You have reached the maximum number (7) of pending unbonding actions"